tests/foreman/api/test_remoteexecution.py

```diff
@@ -50,7 +47,7 @@ def test_positive_run_capsule_upgrade_playbook(module_capsule_configured, target
         data={
             'job_template_id': template_id,
             'inputs': {
-                'target_version': CAPSULE_TARGET_VERSION,
+                'target_version': '',
```

Comment:
Remove the line completely

Reply:
But there also need to be product changes - user option to specify `upgrade` or `update`

Reply:
and `upgrade` is going to always fail for stream/6.16 since there is no 6.17 maintenance bits

Reply:
According to https://github.com/theforeman/foreman_ansible/pull/727 you should also change `job_template_id` from `upgrade` to `update`  unless you provide next version maintenance repo